libsplinter/src/biome/key_management/mod.rs

```diff
@@ -14,6 +14,8 @@
 
```

Comment:
Commit message mentions the `/biome/profile` route, seems to be copy pasta from previous commit

Reply:
Fixed